ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:                                                                                
+  operator-registry:                                                                        
+    name: "4.3"                                                                             
```

Comment:
4.4?

Reply:
yup, updating. I'm also going to leave this commented out until we begin using it soon.